classes/index.php

```diff
@@ -18,16 +19,12 @@
 * needs please refer to http://www.prestashop.com for more information.
 *
 *  @author PrestaShop SA <contact@prestashop.com>
-*  @copyright  2007-2015 PrestaShop SA
+*  @copyright  2007-2019 PrestaShop SA
 *  @license    http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
 *  International Registered Trademark & Property of PrestaShop SA
-*}
-{if $elements}
-  <div id="block-reassurance">
-    <ul>
-      {foreach from=$elements item=element}
-        <li><img src="{$element.image}" alt="{$element.text|escape:'quotes'}" /> <span>{$element.text}</span></li>
-      {/foreach}
-    </ul>
-  </div>
-{/if}
+*/
+
+require(dirname(__FILE__).'/../../config/config.inc.php');
```

Comment:
Why include this ? Use standard index.php please

Reply:
Yes I think it's a mistake, the `index.php` file is supposed to protect the folder
Replace with the class header + exit code